lib/python/mod_python/psp.py

```diff
@@ -271,7 +271,7 @@ def __init__(self, label, file, cache):
                     psp.error_page.run({"exception": (et, ev, etb)}, flush)
                 else:
                     if PY2:
-                        raise et, ev, etb
+                        exec('raise et, ev, etb')
```

Comment:
This does not seem right to me: if this is Python3, then why are we in the `if PY2` block in the first place? This problem is about `PY2` not being set correctly I suspect?